.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "third_party/json"]
 	path = third_party/json
 	url = https://github.com/nlohmann/json
+[submodule "third_party/onnxruntime"]
```

Comment:
Do you think we will use more components in onnxruntime? I guess we'll unlikely do? If it's the case, I'd prefer not submoduling it. As onnxruntime is fairly big (especially with recursive clone), I'm afraid it'd slow down CI or developers. Also, I'm not sure, but I'm guessing their plan is to make the quantize module available from pip (probably by contributing it to ONNX?).

Reply:
I'm concerning that too.
Currently downloading [quantize.py](https://github.com/microsoft/onnxruntime/blob/2698edbc98950c31edbae91af8ca049f61c46eb6/onnxruntime/python/tools/quantization/quantize.py) with curl seems enough for now so will fix this.